src/material-experimental/mdc-progress-spinner/progress-spinner.ts

```diff
@@ -91,12 +91,12 @@ export class MatProgressSpinner extends _MatProgressSpinnerMixinBase implements
   // TODO: switch to class when MDC removes object spread in foundation
   // https://github.com/material-components/material-components-web/pull/6256
   private _adapter: MDCCircularProgressAdapter = {
-    addClass: (className: string) => this._elementRef.nativeElement.classList.add(className),
-    hasClass: (className: string) => this._elementRef.nativeElement.classList.contains(className),
-    removeClass: (className: string) => this._elementRef.nativeElement.classList.remove(className),
-    removeAttribute: (name: string) => this._elementRef.nativeElement.removeAttribute(name),
+    addClass: (className: string) => this._rootElement.classList.add(className),
+    hasClass: (className: string) => this._rootElement.classList.contains(className),
+    removeClass: (className: string) => this._rootElement.classList.remove(className),
+    removeAttribute: (name: string) => this._rootElement.removeAttribute(name),
     setAttribute: (name: string, value: string) =>
-      this._elementRef.nativeElement.setAttribute(name, value),
+      this._rootElement.setAttribute(name, value),
```

Comment:
Rather than having to introduce another layer in the DOM and move styles around, we could change this method so that it doesn't set the `aria-valuenow`. E.g.

```
setAttribute: (name, value) => {
  if (name !== 'aria-valuenow') {
    this._elementRef.nativeElement.setAttribute(name, value)
  }
}

```

Reply:
+1, that would be a better approach here. Separately, though, we should ask mdc why they use 0 to 1 instead of 0 to 100 (as percent).

Reply:
should we also do the same for mdc progress bar?

Reply:
If it has the same issue then yes. Both have the same accessibility setup.